plugins/org.wso2.developerstudio.eclipse.gmf.esb.diagram/src/org/wso2/developerstudio/eclipse/gmf/esb/diagram/navigator/EsbNavigatorContentProvider.java

```diff
@@ -5846,1007 +5896,749 @@ public void init(ICommonContentExtensionSite aConfig) {
         }
 
         case MediatorFlowEditPart.VISUAL_ID: {
-            return mediatorFlowEditPart(view, parentElement);
```

Comment:
why have we deleted these line in this file?

Reply:
I have split the large method into two sub methods. So this line is went into the other method